presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -0,0 +1,197 @@
+`Iceberg Table Spec <https://iceberg.apache.org/spec/>`_.
+
+The Iceberg connector supports:
+
+* All Presto types, including TIME, TIMESTAMP, DECIMAL
+* CREATE TABLE and CREATE TABLE AS
+* Partitioning tables by any Presto type
+* Dropping tables
+* Renaming tables
+* Deletion by partition
+* Table and column statistics
+* Table and column comments
+* Rolling back a table to a previous state
+* Schema evolution
+
+The Iceberg table state is maintained in metadata files. All changes to table state
+create a new metadata file and replace the old metadata with an atomic swap.
+The table metadata file tracks the table schema, partitioning config,
+custom properties, and snapshots of the table contents.
+
+Iceberg data files can be stored in either Parquet or ORC format, as
+determined by the ``format`` property in the table definition.  The
+table ``format`` defaults to ``ORC``.
+
+Iceberg is designed to improve on the known scalabilty limitations of Hive.  Hive
+stores table metadata in an object store backed by a database like MySQL.  It tracks
+partition locations in the metastore, but not individual data files.  Presto queries
+using the Hive connector must first call the metastore to get partition locations,
+then call the underlying filesystem to list all data files inside each partition,
+and then read metadata from each data file.
+
+Since Iceberg stores the paths to data files in the metadata files, it
+only consults the underlying file system for files that must be read.
+
+Status
+------
+
+The Iceberg connector is not yet supported for production use.
+
+Compatibility With Spark Iceberg
+--------------------------------
+
+Tests have been written to demonstrate that data written by Presto can be read
+using Spark SQL with the Iceberg format, and vice versa.  See
+`TestSparkCompatibility <https://github.com/prestosql/presto/blob/master/presto-product-tests/src/main/java/io/prestosql/tests/iceberg/TestSparkCompatibility.java>`_.
+
+Configuration
+-------------
+
+Iceberg supports the same metastore configuration properties as the Hive connector.
+At a minimum, ``hive.metastore.uri`` must be configured:
+
+.. code-block:: none
+
+    connector.name=iceberg
+    hive.metastore.uri=thrift://localhost:9083
+
+Partitioned Tables
+------------------
+
+Iceberg supports partitioning by specifying transforms over the table columns.
+A partition is created for each unique tuple value produced by the transforms.
+Identity transforms are simply the column name. Other transforms are:
+
+===================================== ================================================================================================
+Transform                             Description
+===================================== ================================================================================================
+``year(ts)``                          The partition value is the integer difference in years between ``ts`` and January 1st. 1970.
```

Comment:
Let's change these to "January 1, 1970" to avoid having a period in the middle.

For all of these, it would be useful to have a friendly explanation first:

> A partition is created for each year.

Reply:
Added.